.github/workflows/ingestion_client.yaml

```diff
@@ -0,0 +1,125 @@
+name: Build Ingestion Client Binaries
+
+on:
+  push:
+    branches: ['main', 'master']
+    paths: ['samples/ingestion/ingestion-client/**', '.github/workflows/ingestion_client.yaml']
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '6.0.x'
+
+      - name: Get Release Versions
+        id: get_release_versions
+        run: |
+          version=$(cat ./samples/ingestion/ingestion-client/Setup/ArmTemplateBatch.json | jq '.variables.Version' | tr -d '"')
```

Comment:
this probably runs under Bash?
consider adding

```
set -u -e -o pipefail
```

at the beginning

Reply:
Fixed!